<a href="https://colab.research.google.com/github/DataTak/text_data_analysis/blob/main/notebook/naver_movie_sentimental_analysis.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# 데이터 불러오기
- 데이터출처: [깃허브](https://github.com/e9t/nsmc/)

In [1]:
import pandas as pd

In [3]:
#데이터가 있는 깃허브 레파지터리에서 불러옴. 구분자는 t로  설정
df = pd.read_csv('https://raw.githubusercontent.com/e9t/nsmc/refs/heads/master/ratings_train.txt', sep='\t')


In [4]:
df.head(1)

id             document  label
0  9976970  아 더빙.. 진짜 짜증나네요 목소리      0

# EDA
- shape
- 레이블 비율

In [5]:
df.shape

(150000, 3)

In [6]:
df.label.value_counts()

label
0    75173
1    74827
Name: count, dtype: int64

In [8]:
#label의 그룹화 및 각 그룹의 비율
df.groupby('label').size() / len(df) *100

label
0    50.115333
1    49.884667
dtype: float64

- 15만개의 영화 댓글 리뷰에 대한 데이터입니다.
- ID, 댓글, 레이블로 구성되어 있습니다.
- 레이블은 부정(0) / 긍정(1) 로 구성되어 있으면 각각의 비율은 약 50.1% / 49.9% 로 구성되어있습니다.